packages/clerk-js/src/core/resources/EmailAddress.ts

```diff
@@ -78,6 +81,45 @@ export class EmailAddress extends BaseResource implements EmailAddressResource {
     return { startEmailLinkFlow, cancelEmailLinkFlow: stop };
   };
 
+  createEnterpriseSsoLinkFlow = (): CreateEnterpriseSsoLinkFlowReturn<
```

Comment:
I believe we use `SSO` (all caps) in names (seeing that for `buildSSOCallbackURL` at least):
```suggestion
  createEnterpriseSSOLinkFlow = (): CreateEnterpriseSSOLinkFlowReturn<
```